src/modals/vm-manage-gpu/ManageGpuModal.js

```diff
@@ -99,7 +99,7 @@ const ManageGpuModal = ({
           key='manage-gpu-select-button'
           variant='primary'
           onClick={onSelect}
-          isDisabled={gpus.length === 0}
+          isDisabled={gpus.length === 0 && !nonExistingSelectedMdevs?.length}
```

Comment:
Only a minor nit-pick.  `gpus.length === 0` is used a bunch, maybe it can just be `const noGpusAvailable = gpus.length === 0` or similar just for a small readability gain.  Same in the other changes.  WDYT?

Reply:
I like the idea to extract it to a separate constant. To avoid using !noGpusAvailable, I created gpusAvailable constant and used that.